tickets/src/components/content/CreateContent.js

```diff
@@ -124,7 +125,7 @@ export class CreateContent extends Component {
                     />
                     <Text>
                       Don’t have a lock? <br />
-                      <Cta href="https://unlock-protocol.com" target="_blank">
+                      <Cta href={config.unlockAppUrl} target="_blank">
```

Comment:
```suggestion
                      <Cta href={config.unlockAppUrl} target="_blank" rel="noopener noreferrer">
```

Wouldn't hurt to add these. Would there be any advantage to using a `Next/Link` here instead of a bare `<a>`?